docs/reference/announcements/870.md

```diff
@@ -0,0 +1,136 @@
+
+Camunda 8.7 is scheduled for release on 11 February, 2025.
+
+- [API updates](#api-updates-saasself-managed)
+- [Identity management updates](#identity-management-updates-saasself-managed)
+- [Installation and deployment updates](#installation-and-deployment-updates-self-managed)
+- [Camunda Java client and Camunda Spring SDK](#camunda-java-client-and-camunda-spring-sdk-self-managed)
+
+:::info
+
+The following blog posts also provide more information on the changes included in the Camunda 8.7 release:
+
+- Upcoming API Changes in Camunda 8
+- Introducing Enhanced Identity Management in Camunda 8.7
+- Streamlined Deployment with 8.7
+
+:::
+
+## API updates <span class="badge badge--long" title="This feature affects SaaS">SaaS</span><span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
+
+The 8.7 release includes API updates to support the move to a [Camunda 8 REST API](/apis-tools/camunda-api-rest/camunda-api-rest-overview.md) unified experience.
+
+### Camunda 8 REST API updates
+
+- New Query endpoints (with advanced search filtering) will be added for process entities (processes, decisions, user tasks, and forms). These will replace the component APIs (Tasklist, Operate) going forward.
+- New endpoints will allow you to manage and query users and resource permissions in an orchestration cluster.
+- All the Camunda 8 REST API endpoints will support resource-based authorizations to enable fine-grained permissions.
+- API terminology is aligned so technical assets have an identical, easily-understood, descriptive property name.
+
+### Deprecated: Operate and Tasklist REST APIs
+
+The deprecation process for the [Operate](/apis-tools/operate-api/overview.md) and [Tasklist](/apis-tools/tasklist-api-rest/tasklist-api-rest-overview.md) REST APIs starts with the 8.7 release. You can begin migrating to the [Camunda 8 REST API](/apis-tools/camunda-api-rest/camunda-api-rest-overview.md) for querying to prepare for this change.
+
+- Version 8.7, 8.8: These APIs are available, but not recommended for new implementations.
+- Version 8.9: These APIs will be deprecated and removed.
+
+### Deprecated: Job-based User Tasks querying
+
+As `Job-worker` user tasks will be deprecated in Camunda 8.9, Camunda recommends you start using `Camunda User Tasks` (formerly known as `Zeebe User Task`) in your process definitions.
+
+- Version 8.7, 8.8: `Job-worker` user tasks are available for querying, but Camunda modelers automatically apply the `Camunda user task` and show a warning message for each job worker user task.
+- Version 8.9: `Job-worker` user tasks will be deprecated. With Camunda 8.9 and later, customers can use the `Job-worker` implementation of user tasks as standard jobs with headers to enable open architecture and composable solutions.
+
+### Deprecated: Zeebe gRPC API endpoints
+
+Several [Zeebe gRPC](/apis-tools/zeebe-api/grpc.md) endpoints will be deprecated.
```

Comment:
Can we rephrase to clarify and avoid confusion?
"With Camunda 8.7 release, we will announce depreciation of several grps endpoints that will be removed in 8.9."
It could be read in to that 8.7 already remove some endpoints or even all grps endpoints.

Reply:
Added 👍